aws/tools/cleanup-ssm/README.md

```diff
@@ -0,0 +1,71 @@
+# cleanup-ssm
+
+A Rust CLI tool for cleaning up old AWS Systems Manager (SSM) parameters.
+
+## Overview
+
+This tool helps you identify and delete SSM parameters that are older than a specified number of days. It's designed to help maintain clean AWS environments by removing outdated parameters while providing safety features like dry-run mode.
+
+## Features
+
+- **Dry-run mode**: Preview which parameters would be deleted without actually deleting them (enabled by default)
+- **Age-based filtering**: Delete parameters older than a specified number of days
+- **Batch processing**: Efficiently processes parameters in batches
+- **Progress reporting**: Shows detailed results of the cleanup operation
+- **Multi-region support**: Specify which AWS region to operate in
+
+## Usage
+
+### Basic usage (dry-run mode)
+```bash
+cargo run
+```
+
+### Delete parameters older than 7 days in us-west-2
+```bash
+cargo run -- --region us-west-2 --older-than 7 --dry-run false
+```
+
+### Command-line options
+
+- `--region` (default: `us-east-1`): AWS region to run the cleanup in
+- `--dry-run` (default: `true`): If true, will not delete any parameters, just show what would be deleted
+- `--older-than` (default: `1`): Number of days - parameters older than this will be deleted
```

Comment:
Is it generally safe to delete params older than 1 day?  Is that the most common operation expected, and won't harm pet instances?

Just making sure that the default set here is a very safe number